### DISCUSSION TITLE
Access denied to FHIR Connector extension

### DISCUSSION THREAD

**Discussion by Mo-la-machette** [nextgenhealthcare/connect#4785] [Q&A] [answered]
Hello, 

I have problems to retrieve FHIR Extensions since one week.
When i go here to download it : https://www.nextgen.com/products-and-services/integration-engine

The button to download the extensions has disappeared
And when i try to access to url for downloading the FHIR Extensions : http://downloads.mirthcorp.com/fhir/3.11.0/fhir-3.11.0.b1424.zip

I have an AccessDenied response
Is it a bug or fhir extensions aren't available anymore ?

Thanks

**Comment by pacmano1** [comment, marked as answer]
This extension was beta and is now part of paid licenses.

This was confirmed today by NG but will defer to their update of course!

I would contact sales.

**Reply by Mo-la-machette** [reply]
Thanks for informations !
